### ISSUE TITLE
D loss function problem

### ISSUE THREAD

**Issue by chaunceysong** [batsa003/salgan#12]
`inp_d = torch.cat((batch_img,batch_map),1)`
`outputs = discriminator(inp_d).squeeze()`
`d_loss = torch.sum(torch.log(outputs))`
Why use ‘batch_map’ here instead of ' -log(dis(real))+log(dis(fake))'？
